clients/privacy-center/constants/index.ts

```diff
@@ -2,8 +2,9 @@
 
 import { Config, IdentityInputs } from "~/types/config";
 import configJson from "~/config/config.json";
+import testConfigJson from "~/config/test-config.json";
 
-export const config: Config = configJson;
+export const config: Config = process.env.NEXT_PUBLIC_APP_ENV === "test" ? testConfigJson : configJson;
```

Comment:
Blah, yeah, I see the issue here... there's not a very clean & maintainable way to wire up a variety of different config.json permutations for Cypress testing, since this kind of configuration is done at build-time. 

What you've done here technically works, but at what cost? I'm not sure if the juice is worth the squeeze here because this just feels like another JSON to maintain and forget about... 🤔

Reply:
OK, so a much cleaner way to do this is to mutate the Redux store at runtime, ala this Cypress test: https://github.com/ethyca/fides/blob/main/clients/privacy-center/cypress/e2e/consent.cy.ts#L77

This requires a lot more refactoring to do, though, and clearly my scope creep has gotten out of control on this PR. I think we should just revert these new Cypress tests, manually test this change works as intended, and move forward! Sorry!

Reply:
New ticket here- https://github.com/ethyca/fides/issues/3025